public/locales/de/translation.json

```diff
@@ -0,0 +1,152 @@
+  },
+  "MenuBars": {
+    "markAsDone": "Als fertig markieren",
+    "unmarkAsDone": "Als nicht fertig markieren",
+    "addImage": "Füge Bild hinzu",
+    "addSticker": "Füge Sticker hinzu",
+    "settings": "Einstellungen",
+    "editColumns": "Bearbeite Spalten",
+    "stopEditingColumns": "Beende Spaltenbearbeitung",
+    "startVoting": "Starte Abstimmung",
+    "stopVoting": "Beende Abstimmung",
+    "startFocusMode": "Starte Fokus-Modus",
+    "stopFocusMode": "Beende Fokus-Modus"
```

Comment:
I am not sure if we should add articles in German or leave it like this to be similar to the English version

Reply:
It'll be too long for our menu bars 😬

Reply:
Yes you are right. So we can leave it like this 👍